### PR TITLE
Updated change and delete implementaion.

### PR DESCRIPTION
**Pull request by teddytwin**

### CODE REVIEW

**Comment by teddytwin** [comment]
[Sak - Endre på utredere](https://trello.com/c/Xo8H57Fr/13-sak-endre-p%C3%A5-utredere)

File: src/main/kotlin/no/nav/dsop/kontroll/service/sak/Routes.kt [hunk 1]
```diff
@@ -90,22 +93,51 @@ fun Route.sak(sakService: SakService) {
...
-                    val saksbehandler: String =
-                        call.getFromRequest(ParamType.Header, HttpHeaders.NavSaksbehandlerId)
-                    val currentUser = call.getAuthenticatedUser()
-                    when (sakService.getSak(saksnummer)) {
-                        null -> call.respond(HttpStatusCode.NotFound, "No sak with saksnummer '$saksnummer' found.")
-                        else -> {
-                            sakService.insertSakUtreder(currentUser, saksbehandler, saksnummer)
-                            call.respond(HttpStatusCode.Created)
-                        }
+                    val nyUtreder = objectMapper.readValue<AuthenticatedUser>(call.receiveTry<String>())
```
Comment (tronghn) [inline src/main/kotlin/no/nav/dsop/kontroll/service/sak/Routes.kt]: Could be simplified to `call.receiveTry<AuthenticatedUser>()`

File: src/main/kotlin/no/nav/dsop/kontroll/service/sak/Routes.kt [hunk 2]
```diff
@@ -90,22 +93,51 @@ fun Route.sak(sakService: SakService) {
...
+                                        )
+                                    )
+                                }
+                            } catch (e: Throwable) {
+                                Result.failure<SakResponse?>(IllegalStateException("Oisann! Her har noe gått galt."))
+                            }
+                    }
+                    call.respond(HttpStatusCode.Accepted, result.toResource())
+
+/*
```
Comment (tronghn) [inline src/main/kotlin/no/nav/dsop/kontroll/service/sak/Routes.kt]: Might as well remove the commented code below if the above is replacing it?

File: src/test/kotlin/no/nav/dsop/kontroll/services/SakServiceSpek.kt [hunk 3]
```diff
@@ -230,6 +236,32 @@ object SakServiceSpek : Spek({
...
+                    context("DELETE last utreder from sak") {
+                        with(handleRequest(HttpMethod.Delete, "/api/v1/sak/${request.saksnummer}/utreder") {
+                            addHeader(HttpHeaders.NavSaksbehandlerId, nyUtreder.identifier)
+                        }) {
+                            val expectedStatusCode = HttpStatusCode.Accepted
+                            it("should return $expectedStatusCode") {
+                                response.status() shouldEqual expectedStatusCode
+                            }
+                            it("should return a valid response") {
+                                val status =
```
Comment (tronghn) [inline src/test/kotlin/no/nav/dsop/kontroll/services/SakServiceSpek.kt]: Unused? Could also simplify by reusing the deserialized object for the assertions. I'd also recommend to not hard-code assertions on strings that may change in the future

**Pull request merged**